elastic-stack/README.md

```diff
@@ -0,0 +1,6 @@
+# Elastic Stack on GKE
```

Comment:
Would you mind moving this tutorial to either `monitoring/elastic-stack-tutorial/` or `operations/elastic-stack-tutorial/`? I'm trying to get away from single samples as top-level directories, just for clean-up / sustainability purposes